<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dijkstra Visualization</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="GraphArea" name="graphArea"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="drawVertexButton">
        <property name="text">
         <string>Draw Vertex</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="drawEdgeButton">
        <property name="text">
         <string>Draw Edge</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="randomGenerateButton">
        <property name="text">
         <string>Rand Generate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stepButton">
        <property name="text">
         <string>Step</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="showPathButton">
        <property name="text">
         <string>Show Path</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>859</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
    <addaction name="actionDebug_only"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout_Me">
   <property name="text">
    <string>About Me</string>
   </property>
  </action>
  <action name="actionDebug_only">
   <property name="text">
    <string>Debug only</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GraphArea</class>
   <extends>QGraphicsView</extends>
   <header>GraphArea.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
